src/Processors/QtiV2/In/Interactions/TextEntryInteractionMapper.php

```diff
@@ -25,6 +47,35 @@ public function getQuestionType()
         return $closetext;
     }
 
+    public function getOption($key, $defaultValue = null) {
+        $value = $defaultValue;
+        if (isset($this->options[$key])) {
+            $value = $this->options[$key];
+        }
+
+        return $value;
+    }
+
+    public function isUsingExpectedLengthAsMaxCharLimit() {
+        return (bool)$this->getOption('expected_length_as_max_char_limit', false);
+    }
+
+    public function isUsingExpectedLengthAsResponseWidth() {
+        return (int)$this->getOption('expected_length_as_response_width', true);
+    }
+
+    public function setOption($key, $value) {
+        $this->options[$key] = $value;
+    }
+
+    public function setUsingExpectedLengthAsMaxCharLimit($value = false) {
+        $this->setOption('expected_length_as_max_char_limit', (bool)$value);
+    }
+
+    public function setUsingExpectedLengthAsResponseWidth($value = true) {
```

Comment:
The default `$value` is a boolean while we coerce it to an int when setting. Should we just use a different default?

Reply:
I'm going to remove these individual setters for now as we're not likely to be configuring mappers individually at this level - if anything the options will be defined in a configuration object and passed down wholesale.